### PR TITLE
Draftt 49 auth navigator

### PR DESCRIPTION
**Pull request by aarajh**
Added an auth navigator with the basic structure required to build on. Once you log in, you cannot go back to the login screens on pressing the back button. Once we remove the links from tester screen, home will be inaccessible without logging in.

### CODE REVIEW

File: pitch/src/screens/Home/home.enhancer.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import { connect } from 'react-redux';
 import {compose} from 'recompose';
 
+
```
Comment (adil1508) [inline pitch/src/screens/Home/home.enhancer.js:5]: @aarajh , what is this behavior?

File: pitch/src/screens/Home/home.js [hunk 2]
```diff
@@ -15,6 +15,16 @@ const Home = () => {
 				/>
 			</View>
 			<Text style={styles.titleTextStyle}>Home Screen</Text>
+			<TouchableOpacity
+							style={styles.buttonStyle}
+							onPress={() => {
+								navigation.navigate('Tester');
+							}}>
+							<Text style={{ fontSize: 15, color: "white" }}>
+								{" "}
```
Comment (adil1508) [inline pitch/src/screens/Home/home.js:24]: ???

File: pitch/src/styles/navigation.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { createAppContainer } from "react-navigation";
```
Comment (adil1508) [inline pitch/src/styles/navigation.js:1]: @aarajh , what is this file man 🤦‍♂️

File: pitch/src/screens/Home/home.enhancer.js (hunk 1 above)
Comment (aarajh) [reply pitch/src/screens/Home/home.enhancer.js:5]: ...

File: pitch/src/screens/Home/home.js (hunk 2 above)
Comment (aarajh) [reply pitch/src/screens/Home/home.js:24]: YOU WROTE THIS CODE.

File: pitch/src/styles/navigation.js (hunk 3 above)
Comment (aarajh) [reply pitch/src/styles/navigation.js:1]: Vscode trying to be smart by auto-importing and creating files? :p

**Comment by adil1508** [comment]
Going to rebase after middleware branch is merged

File: pitch/src/screens/Home/home.js (hunk 2 above)
Comment (adil1508) [reply pitch/src/screens/Home/home.js]: WELL THEN DONT WILLY NILLY COMMIT IT AGAIN. THAT WAS A TEST

**Pull request merged**